### ISSUE TITLE
Importing Builds: Import Exception importing build from game export

### ISSUE THREAD

**Issue by mutaku** [MidwestSteeze/ChroniCalc#1]
Error:  https://imgur.com/t3ApdUa
```Could not convert string to integer: 
Path 'metadata', line 1, position 1495```

Details: 
```---------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 12.0.0.0
    Win32 Version: 12.0.2.23222
    CodeBase: file:///C:/Users/matthew/AppData/Local/Apps/2.0/DXXEML47.B4T/TTVD3QL4.8X2/chro..tion_ea2646810ba28a1f_0000.0009_b5839e4920a34911/Newtonsoft.Json.DLL
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
System.Runtime.Serialization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_32/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by MidwestSteeze** [comment]
- changed build import process to read the .build file purely as strings to allow for additional Key/Value pairs in the .build that aren't actually Skill IDs and their respective level (e.g. "metadata"); this should make the .build data import a little more flexible incase more non-Skill ID data is added to it in the future
 - fixed issue importing Mastery skills with regard to how they are stored in the .build file vs how they're read into the program (I was previously assuming all Skill IDs in the .build file had an "s" variant, which was me expecting two entries for each skill in the .build file; one which contained the level and the other which contained the Slot ID; now, only "shared/generic" mastery skills contain an extra entry with its corresponding Slot ID)
 - fixed the build import process specific to importing Mastery skills; tree counter skills (ie. the ones on the far left of the trees) were being incorrectly incremented, as well as the build level and total mastery level

TLDR @mutaku Sorry for the delay but I'm back at it and this should now be fixed.

**Comment by mutaku** [comment]
> 
> 
>     * changed build import process to read the .build file purely as strings to allow for additional Key/Value pairs in the .build that aren't actually Skill IDs and their respective level (e.g. "metadata"); this should make the .build data import a little more flexible incase more non-Skill ID data is added to it in the future
> 
>     * fixed issue importing Mastery skills with regard to how they are stored in the .build file vs how they're read into the program (I was previously assuming all Skill IDs in the .build file had an "s" variant, which was me expecting two entries for each skill in the .build file; one which contained the level and the other which contained the Slot ID; now, only "shared/generic" mastery skills contain an extra entry with its corresponding Slot ID)
> 
>     * fixed the build import process specific to importing Mastery skills; tree counter skills (ie. the ones on the far left of the trees) were being incorrectly incremented, as well as the build level and total mastery level
> 
> 
> TLDR @mutaku Sorry for the delay but I'm back at it and this should now be fixed.

@MidwestSteeze Don't apologize at all! Looks great, thanks so much for all your hard work - you're providing great value to the Chronicon community.
